tools/topology/sof-hsw-nocodec.m4

```diff
@@ -0,0 +1,97 @@
+#
+ifelse(PLATFORM, `hsw', `# Topology for generic CherryTrail board with no codec.', `')
+ifelse(PLATFORM, `bdw', `# Topology for generic BayTrail board with no codec.', `')
```

Comment:
What do Baytrail and cherrytrail do here?

Reply:
@plbossart copy paste from the working sof-byt-nocodec.m4, will fix.